src/vs/workbench/contrib/notebook/browser/view/cellParts/cellEditorOptions.ts

```diff
@@ -86,6 +86,19 @@ export class CellEditorOptions extends Disposable {
 		this._value = this._computeEditorOptions();
 	}
 
+	prepareRender(): void {
```

Comment:
It could be optional to implement these, seems that parts often don't need to

Reply:
not everyone needs `updateLayoutNow` either so we can probably use `override` other than `abstract`

Reply:
I might still keep it though to ensure that when we add a new CellPart, we are aware that batching DOM read and writes are necessary.